site/ja/guide/create_op.md

```diff
@@ -0,0 +1,1231 @@
+# オペレーションを作成する
+
+Note: C++のカスタムオペレーションが、TensorFlow公式のpipパッケージとABI互換になることを保証するため、[Custom opリポジトリ](https://github.com/tensorflow/custom-op) のガイドにしたがってください。
+カスタムオペレーションをビルドして配布するためのDockerのイメージはもちろんのこと、端から端までコード例が示されています。
```

Comment:
[weak suggestion]
```suggestion
カスタムオペレーションをビルドして配布するためのDockerのイメージはもちろんのこと、初めから終わりまでのコード例が示されています。
```

Reply:
Thanks for your suggestion. I used this expression.